### DISCUSSION TITLE
Hitting mute when muted doesn’t unmute on LG 27UL650-W

### DISCUSSION THREAD

**Discussion by leana8959** [MonitorControl/MonitorControl#883] [Q&A] [answered]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Looked through [the wiki](https://github.com/MonitorControl/MonitorControl/wiki)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the bug

When the audio is muted or dropped to minimum, hitting mute again doesn't unmute, I have to use the monitor's control to unmute.

### Steps to reproduce

- Press two times mute
- Lower the volume to minimum and then raise the volume

### Expected behavior

Be able to unmute with the unmute key.

### Environment Information (please complete the following information)

- macOS version: 12.1 Monterey
- Mac model: MacBook Pro (M1, 2020)
- MonitorControl version: v4.0.2
- Monitor(s): LG 27UL650-W
- Apple Silicon : Yes

**Comment by waydabber** [comment, marked as answer]
You can try toggling the "Enable Mute DDC command" under Displays/Show advanced preferences.

**Reply by leana8959** [reply]
It works now ! I didn't look into the advanced settings since it's hidden.
Thank you and have a nice day !

**Reply by waydabber** [reply]
Great. This issue affects a limited amount of displays, that's why it is that hidden. :)
